webapp/src/components/ImageFieldEditor.tsx

```diff
@@ -6,6 +6,7 @@ import { setTelemetryFunction } from './ImageEditor/store/imageReducer';
 
 export interface ImageFieldEditorProps {
     singleFrame: boolean;
+    parentCallback?: () => void;
```

Comment:
use a more descriptive name--"doneButtonCallback" or "onDoneButtonClicked", for example